doc/layers.md

```diff
@@ -1,12 +1,18 @@
 # Layers
 
-Layers are an useful feature that lets you group renderization by context, as well as allow you to pre-render things. That enables, for example, the renderization in memory of parts of your game that don't change much, like a background, and by doing that, freeing resources for more dynamic content that needs to be rendered every loop cycle.
+Layers are an useful feature that lets you group rendering by context, as well as allow you to
+pre-render things. This enables, for example, rendering parts of your game that don't change much in
+memory, like a background. By doing this, you'll free resources for more dynamic content that needs
+to be rendered every loop cycle.
 
-There are two types of layers on Flame: `DynamicLayer` (for things that are moving or changing) and `PreRenderedLayer` (for things that are static).
+There are two types of layers on Flame: `DynamicLayer` (for things that are moving or changing) and
+`PreRenderedLayer` (for things that are static).
 
 ## DynamicLayer
 
-Dynamic layers are layers that are rendered every time that they are draw on the canvas. As the name suggests, it is meant for dynamic content and is most useful to group renderizations that are of the same context.
+Dynamic layers are layers that are rendered every time that they are draw on the canvas. As the name
```

Comment:
```suggestion
Dynamic layers are recomputed every time that they are drawn on the canvas. As the name
```

Reply:
Added `drawn`, but not convinced by `recomputed`, I think that is a bit unclear?

Reply:
Because it is the actual rendering that is grouped, right?